tools/CAT/src/pages/pages/components/DialogCreateCategory/index.tsx

```diff
@@ -1,16 +1,22 @@
-import { useState, useEffect } from "react";
+import { useState } from "react";
 import Dialog from "@/components/Dialog";
 import Button from "@/components/Button";
 import { useAddCategoryMutation } from "@/store/apis/categories";
-import TextField from "@/components/TextField";
+import Edit from "@/components/Edit";
 
-interface PropsCategory {
+interface StateCategory {
 	title: string;
 	url: string;
 }
-
-export default function DialogCreateCategory({ handleClose, data }) {
-	const [createCategory, setCreateCategory] = useState<PropsCategory>({
+interface PropsCategory {
```

Comment:
Please check this article

https://levelup.gitconnected.com/managing-types-in-react-typescript-the-right-way-fa1ecc50a2bf